qiskit_machine_learning/kernels/algorithms/__init__.py

```diff
@@ -0,0 +1,31 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Quantum Kernel Algorithms (:mod:`qiskit_machine_learning.kernels.algorithms`)
```

Comment:
Quantum Kernel Algorithms - QKA? or is that Quantum Kernel Alignment. If the latter maybe this is Quantum Kernel algorithms (lowercase) to avoid confusion. In relation to the runtime do we have a local QKA , whatever that is - or is that part of the QSVC by virtue of adding the trainer?